base/bucket_gocb.go

```diff
@@ -353,8 +353,12 @@ func (bucket *CouchbaseBucketGoCB) Get(k string, rv interface{}) (cas uint64, er
 	}
 
 	// Kick off retry loop
-	description := fmt.Sprintf("Get %v", k)
-	err, result := RetryLoop(description, worker, bucket.spec.RetrySleeper())
+	err, result := RetryLoop("Get", worker, bucket.spec.RetrySleeper())
+	if err != nil {
+		if _, ok := err.(*RetryTimeoutError); ok {
```

Comment:
How come we're checking for only this error now? Shouldn't it be kept as all errors?

Reply:
I wanted to ensure the same behaviour was kept as some callers are checking for error types. I'll double check. Although it looks like `IsDocNotFoundError` check should handle wrapped errors

Reply:
Fair enough - didn't even consider consumers of the error, but I agree it makes sense to have stuff further up the stack unwrap if nessesary.

Reply:
Same with `IsKeyNotFoundError`